build/images/scripts/start_ovs

```diff
@@ -109,6 +109,12 @@ SLEEP_PID=
 
 function quit {
     log_info $CONTAINER_NAME "Stopping OVS before quit"
+    # sleep up to 10 seconds to wait antrea-agent shutdown for AntreaIPAM
```

Comment:
I am not sure it is an issue to always wait for 10 more seconds before antrea-ovs exits. @antoninbas @tnqn

Reply:
I feel like we don't have a choice, we need to make sure the agent is done before stopping ovs to ensure proper cleanup. As long as users don't configure a shorter grace period (or no grace period at all) for the Pod, it should be fine. But:

* I don't see a sleep right now, it seems that we call the command 10x in a row without sleeping. Am I missing something?
* Is there a more elegant way to check that the antrea-agent process exited?
* Maybe a 10s timeout is arbitrary. IMO, we should keep waiting until either the agent is done or we get the sigkill from kubelet (end of the grace period)